src/main/java/org/killbill/billing/payment/api/Payment.java

```diff
@@ -84,4 +84,9 @@
      * @return the list of attempts on that payment
      */
     List<PaymentTransaction> getTransactions();
+
+    /**
+     * @return the next Scheduled payment attempt
+     */
+    PaymentAttempt getNextScheduledPaymentAttempt();
```

Comment:
@pierre 
I'm not sure about expecting a **List** of Payment Attempts, since in my code [here](https://github.com/javier-gomez-hs/killbill/blob/b657099c18e3da02216f4e17be86fa18b86015ce/payment/src/main/java/org/killbill/billing/payment/core/PaymentProcessor.java#L479-L479) I'm just getting the **next** scheduled Attempt, and not the complete list of future Attempts.
Should I change my method to return the whole set of Future Attempts?